orders/views.py

```diff
@@ -1,3 +1,107 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http     import JsonResponse
+from django.views    import View
+
+from users.models    import Destination
+from products.models import Product, ProductOption
+from .models         import Order, Cart, Status
+from users.utils     import login_required
+from orders.utils    import refresh_order, query_debugger
+
+class CartView(View):
+    @query_debugger
+    @login_required
+    def post(self, request):
+        try:
+            data           = json.loads(request.body)
+            quantity       = data['quantity']
+            product        = Product.objects.get(id=data['product'])
+            user           = getattr(request,'user',None)
+            before_execute = Status.objects.get(id=1)
+            option         = product.productoption_set.get(id=data['option'])
+
+            if getattr(user.order_set.last(),'status',None) != before_execute:
+                Order.objects.create(
+                    user=user,
+                    shipping_memo="",
+                    total_price=0,
+                    status=before_execute
+                )
+            order = Order.objects.last()
```

Comment:
```suggestion
                order = Order.objects.create(
                    user=user,
                    shipping_memo="",
                    total_price=0,
                    status=before_execute
                )
```
확률이 낮긴 하지만, 동시에 여러 주문이 들어올 경우, 방금 생성한 내 주문과 가장 마지막 주문이 동일할 것이라고 보장하기 어렵습니다. 따라서 주문을 생성함과 동시에 변수에 할당하는 방법을 추천드립니다